### ISSUE TITLE
Option to change Modal Presentation Style when starting verification

### ISSUE THREAD

**Issue by pietrobasso** [idnow/de.idnow.ios#73]
## Current situation
Starting from iOS 13, IDNow is presented modally as a `pageSheet` (which is the new default behavior for modally presented view controllers). **IDNow flow can be dismissed by swiping down on the presented screen.**

![image](https://user-images.githubusercontent.com/2761772/70425585-c6f82800-1a71-11ea-9a1d-54f83e86a6b1.png)

## Proposed solution

You should add the option to choose which modal presentation style should `IDNow` use when starting verification.

```swift
func startIdentification(from: UIViewController, withCompletionBlock: IDnowCompletionBlock, modalPresentationStyle: UIModalPresentationStyle) { ... }
```

## Reasoning behind this
In our current flow, user can terminate their IDNow session tapping on the X button on the upper left corner. User shouldn't terminate their IDNow session by swiping down the modal presented screen. We want to preserve `fullScreen` modal presentation style from iOS 12.

**Comment by phu-idnow** [comment]
hi @pietrobasso ,
You can set your desire type via IDNowSettings `modalPresentationStyle`
for example:

`settings.modalPresentationStyle = .fullScreen
let idnController = IDnowController(settings: settings)
`

Thanks,

**Comment by pietrobasso** [comment]
Hi @phu-idnow, thank you for the quick response. 
I tried to set `modalPresentationStyle` on IDNowSettings, but this setting has no effect on the presentation style.
See https://github.com/idnow/de.idnow.ios/issues/63 for reference.

**Comment by phu-idnow** [comment]
hmmm, very strange. which SDK version are you using?

Thanks

**Comment by phu-idnow** [comment]
can you please share your `IDNowSettings`?

**Comment by pietrobasso** [comment]
I'm using version `3.17.0`. Could you please point out which version was this bug fixed in?
The `IDNowSettings` object is just an empty one, no additional parameters are set aside from `modalPresentationStyle`.

**Comment by phu-idnow** [comment]
hi @pietrobasso , We fixed in 3.21.0, but some improvements you should give a try with the latest one (4.0.0)

Thanks,

**Comment by pietrobasso** [comment]
Small heads up, updating to `3.22.0` solved the issue. No luck with version `3.21.0`. Going to close this ticket. Thanks @phu-idnow for helping out!
